libshaderc/src/shaderc_test.cc

```diff
@@ -52,6 +52,14 @@ TEST(Init, MultipleThreadsCalling) {
   shaderc_compiler_release(compiler3);
 }
 
+TEST(Init, SPVVersion) {
+  unsigned int version = 0;
+  unsigned int revision = 0;
+  shaderc_get_spv_version(&version, &revision);
+  EXPECT_EQ(version, spv::Version);
```

Comment:
Yeah, it matters for correct failure messages -- it's pretty nifty when a failing test says something like
```
Value of: version
  Actual: 456
Expected: spv::Version
Which is: 123
```